templates/Includes/BlogEntryTags.ss

```diff
@@ -4,7 +4,7 @@
             <p class="blog-tags">
                 <%-- If you are using the category list below, uncomment the "Tags:" label here --%>
                 <%--<%t BlogEntry.Tags "Tags:" %>--%>
-                <% loop $Tags %><a href="$Link" title="$Title.ATT" class="label label-primary m-r-05">$Title</a><% end_loop %>
+                <% loop $Tags %><a href="$Link" title="$Title.ATT" class="badge badge-primary m-r-2">$Title</a><% end_loop %>
```

Comment:
I think m-r-2 should be mr-2.
m-r-05 and m-r-2 is defined in starter/src/utils/helpers but this file should probably be removed now that bootstrap 4 defines these spacing classes.